src/js/utils/background.js

```diff
@@ -137,13 +182,32 @@ export const backgroundStyle = (backgroundArg, theme, textColorArg) => {
     theme,
   );
 
-  if (background.image) {
+  let backgroundImage = normalizeBackgroundImage(background, theme);
+
+  if (background.rotate) {
+    backgroundImage = rotateBackground(background, theme);
```

Comment:
This looks like we're throwing away what we did at line 185 with `normalizeBackgroundImage()`. Should we change line 185 to be a ternary, i.e. `background.rotate ? rotateBackground() : normalizeBackgroundImage()`

Reply:
Done.